src/main/java/com/booksaw/betterTeams/commands/team/KickCommand.java

```diff
@@ -47,7 +47,7 @@ public CommandResponse onCommand(TeamPlayer teamPlayer, String label, String[] a
 		team.removePlayer(player);
 
 		if (player.isOnline()) {
-			MessageManager.sendMessageF((CommandSender) player, "kick.notify", team.getName());
+			MessageManager.sendMessage((CommandSender) player, "kick.notify", team.getName());
 
 			if (Main.plugin.getConfig().getBoolean("titleRemoval")) {
 				player.getPlayer().sendTitle(" ", MessageManager.getMessage("kick.title"), 10, 100, 20);
```

Comment:
Could you switch existing places there is a title message to the new system

Reply:
I saw this, but the current system is using the subtitle only, not the title. And I was wondering how I could integrate it without rehauling the messaging manager totally 🤣.

Do you have an idea on how to approach this?